cmd/bucket-replication.go

```diff
@@ -2477,6 +2481,8 @@ func replicationResyncTrace(resyncID string, startTime time.Time, duration time.
 		Duration:  duration,
 		Path:      path,
 		Error:     errStr,
+		OpName:    tc.OpName,
+		OpID:      tc.OpID,
```

Comment:
I am wondering if we should add a `map[string]string` for fields that can be displayed as `key=value` on (detailed) trace outputs. This would be much more flexible in the future and we could move some of the stuff like this:

https://github.com/minio/minio/blob/f27f886e5cac672489329b4cd32cb814f118c56d/cmd/erasure-healing.go#L1055